.github/workflows/unittests.yml

```
@@ -100,7 +100,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        tf-version: [1.14.0, 1.15.5, 2.0.4, 2.1.4, 2.2.3, 2.3.3, 2.4.4, 2.5.3, 2.6.4, 2.7.2, 2.8.1, 2.9.0, 2.10.0]
+        tf-version: [1.14.0, 1.15.5, 2.0.4, 2.1.4, 2.2.3, 2.3.3, 2.4.4, 2.5.3, 2.6.4, 2.7.2, 2.8.1, 2.9.0, 2.10.0, 2.11.0]
```

Not related to this PR, but shouldn't we remove some stuff here to save CI time? I understand that in the public version we might want to test a lot, but privately I think this doesn't make much sense, right? E.g.:
```suggestion
        tf-version: [1.15.5, 2.8.1, 2.9.0, 2.10.0, 2.11.0]
```